common.js

```diff
@@ -61,6 +61,7 @@ module.exports = {
     'no-var': 'error',
     'no-return-await': 'error',
     'require-await': 'warn',
+    'spaced-comment': ['warn', 'always'],
```

Comment:
I'd like to see the [`balanced`](https://eslint.org/docs/rules/spaced-comment) option with this as well.

```suggestion
    'spaced-comment': ['warn', 'always', {
      block: { balanced: true }
    }],
```